docs/features/runs/private-repos.md

```diff
@@ -4,21 +4,12 @@ Grid Runs offers two solutions for using private repos:
 1. Grant Grid access to your private GitHub repos
 2. Utilize the grid run --localdir option
 
-## Access Private GitHub Repos
-
-By default you can only access public Github repositories on Grid. To grant read access to your private code navigate to Settings &gt; Integrations &gt; Grant access.
-
-![](/images/platform/grant_github_access.gif)
-
-### About private code
-
-Grid does not save your code, look at it or compromise its privacy in any way.
-
-When receiving support, you will not have to share any code to help debug. If you choose to share code, make sure you have the rights to and share non-critical parts of the code.
+## Integrating with GitHub
+See [here](https://docs.grid.ai/platformgithub-integration) for detailed steps for integrating GitHub with Grid.
```

Comment:
is this the URL you intended?

Reply:
@essiequoi I believe so. I fixed a typo but won't that url be active once the changes are merged into master?